### DISCUSSION TITLE
Manually reorder records

### DISCUSSION THREAD

**Discussion by rmeekers** [Budibase/budibase#12808] [Help]
I'm looking for a way to manually reorder records in a table (preferably through an interface).
The use case is the following: I have a list of tasks, and the user needs to order them in terms of priority.

Andybody an idea how I can accomplish this?

**Comment by melohagan** [comment]
Hey @rmeekers You could take a look at Conor's Kanban plugin component: https://github.com/ConorWebb96/bb-kanban-draggable

**Reply by melohagan** [reply]
Or if it's a list of tasks in a single column, then you could order this list by that numeric priority column, and then use the **Refresh Data provider** action to re-trigger the ordering after a priority has changed.

**Comment by rmeekers** [comment]
> and then use the **Refresh Data provider** action to re-trigger the ordering after a priority has changed.

But how do you update the order from the interface? And avoid duplicate numbers in the priority column?

**Reply by melohagan** [reply]
I think it would help to see some screenshots to get an idea of how you expect your interface to look.

**Reply by rmeekers** [reply]
<img width="1554" alt="Screenshot 2024-01-18 at 13 08 13" src="https://github.com/Budibase/budibase/assets/5755308/b9d29f3f-e455-4d7e-8e67-768b5b1ec42c">

Something like this, where you have arrows on each record to move it a position up or down.

**Reply by melohagan** [reply]
Okay thanks for that @rmeekers ! I'll have a play around and see what I can come up with.

**Comment by rmeekers** [comment]
> Hey @rmeekers You could take a look at Conor's Kanban plugin component: https://github.com/ConorWebb96/bb-kanban-draggable

This plugin only allows to move cards from one column to another, not to change the order within a column. And it's exactly that what I'm looking for.
(but thanks for the suggestion ☺️)
